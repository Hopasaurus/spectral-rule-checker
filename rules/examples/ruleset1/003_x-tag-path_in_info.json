{
  "swagger": "2.0",
  "info": {
    "title": "Sample API",
    "description": "API description in Markdown.",
    "version": "1.0.0",
    "contact": { "email": "email@example.com" },
    "x-tag-info": "example x-tag-info",
    "x-tag-path": "this should cause an error"
  },
  "host": "api.example.com",
  "basePath": "/v1",
  "tags": [
    {
      "name": "sample",
      "description": "sample tag",
      "externalDocs": {
        "description": "external docs description",
        "url": "http://example.com"
      }
    }
  ],
  "schemes": [
    "https"
  ],
  "paths": {
    "/users": {
      "get": {
        "x-tag-path": "sample x-tag-path",
        "tags": ["sample"],
        "operationId": "getUsers",
        "summary": "Returns a list of users.",
        "description": "Optional extended description in Markdown.",
        "produces": [
          "application/json"
        ],
        "responses": {
          "200": {
            "description": "OK"
          }
        }
      }
    }
  }
}