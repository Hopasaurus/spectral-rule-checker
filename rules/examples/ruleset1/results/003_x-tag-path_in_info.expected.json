[
  {
    "code": "no-x-tag-path-in-info",
    "message": "x.y.z - x-tag-info must NOT be in $.info",
    "path": [
      "info",
      "x-tag-path"
    ],
    "severity": 0,
    "range": {
      "start": {
        "line": 8,
        "character": 18
      },
      "end": {
        "line": 8,
        "character": 46
      }
    }
  }
]